attributes/default.rb

```diff
@@ -11,13 +11,18 @@
 default[:hadoop][:tmp_dir]                 = "#{node[:hadoop][:home]}/tmp"
 default[:hadoop][:conf_dir]                = "#{node[:hadoop][:home]}/etc/hadoop"
 default[:hadoop][:sbin_dir]                = "#{node[:hadoop][:home]}/sbin"
-default[:hadoop][:bin_dir]                = "#{node[:hadoop][:home]}/bin"
+default[:hadoop][:bin_dir]                 = "#{node[:hadoop][:home]}/bin"
 default[:hadoop][:data_dir]                = "/var/data/hadoop"
 default[:hadoop][:dn][:data_dir]           = "#{node[:hadoop][:data_dir]}/hdfs/dn"
 default[:hadoop][:nn][:name_dir]           = "#{node[:hadoop][:data_dir]}/hdfs/nn"
 
-default[:hadoop][:download_url]            = "https://archive.apache.org/dist/hadoop/core/hadoop-#{node[:hadoop][:version]}/hadoop-#{node[:hadoop][:version]}.tar.gz"
-#default[:hadoop][:download_url]            = "#{node[:download_url]}/hadoop-#{node[:hadoop][:version]}.tar.gz"
+default[:hadoop][:download_url]            = "ftp://ftp.fu-berlin.de/unix/www/apache/hadoop/common/hadoop-#{node[:hadoop][:version]}/hadoop-#{node[:hadoop][:version]}.tar.gz"
+version = "#{node[:hadoop][:version]}"
+version.gsub!('.', '')
```

Comment:
This is taking recent versions for fu-berlin, and old versions from hadoop archive. Is that correct?